db/migrate/20181126094551_update_all_first_admin_id.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class UpdateAllFirstAdminId < ActiveRecord::Migration
+  def up
+    Account.transaction do
+      Account.providers.find_each do |account|
+        account.update_column(:first_admin_id, account.first_admin&.id) or raise ActiveRecord::Error
```

Comment:
It seems that this update could run separately in script instead of in a blocking migration.

Reply:
Ok about doing it in a separate script but I wouldn't say this is a blocking migration since this blocks only one record at a time 😄